docs/specs/data-model.md

```
@@ -0,0 +1,150 @@
+| `question` | `string` | ユーザーの深掘り質問 |
+| `answer` | `string` | AI の回答（Markdown） |
+
+## RefinementEntry
+
+ブラッシュアップの1エントリ。
+
+| フィールド | 型 | 説明 |
+|-----------|---|------|
+| `review` | `string` | ユーザーのレビューコメント |
+| `answer` | `string` | AI の回答（Markdown） |
+| `results` | `AIResults` | - | 構造化された更新結果（任意） |
```

markdownlint-cli2 (0.21.0)

[warning] Table column count
Expected: 3; Actual: 4; Too many cells, extra data will be missing

(MD056, table-column-count)